src/components/Motorcycles/Motorcycle/Motorcycle.js

```diff
@@ -56,7 +56,10 @@ function Motorcycle(props) {
             <p className={styles.description}>
               {props.description}
             </p>
-            <p className="mt-2">{ auth ? 'Dostępność: 2 motocykle' : 'Dostępność: Zaloguj' }</p>
+            {auth 
+              ? <p className="mt-2">Dostępność: {props.motorcycle} motocykle</p>
+              : <p className="mt-2">Dostępność: zaloguj</p>
+            }

```

Comment:
Use -->` { auth ? '${props.motorcycle} motocykle' :  'zaloguj' }`

in only one p tag. Always try to place conditions in the lowest possible place.

instead of ' please use `. I couldn't use this in the comment

Reply:
Ok right, i should look more closely at your advice